### PR TITLE
<GooglePreview> - Add `titleMaxLines`, `descriptionMaxLines` and `skin` props

### PR DESCRIPTION
**Pull request by tomershk**
### 🔦 Summary

Make the google preview component up to date.

1. Enable options of multi-line in title and description: 
- new props: `titleMaxLines` `descriptionMaxLines`

2. Enable option of transparent BG:
- new prop: `skin` with the options: `default` or `transparent`
- In case value is `transparent` border will also be hidden

3. Change the UI of the component to match the Google existing design:
- Move url to top
- Change url color to according to google.

Link to monday: https://wix.monday.com/boards/535450707/pulses/938373975

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @EgoziE 
- [x] 👨‍🎨 UX change is approved by the Design System UX @ayeletr 
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [x] Visual test

### CODE REVIEW

**Comment by TalZa1810** [comment]
Hi @tomershk 
Can you please [sign your commits](https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md#sign-commits) so that the CI will run?
Thanks

**Review by EgoziE — CHANGES_REQUESTED** [review]
Please add all new props to types & type test file (index.d.ts & GooglePreview.tsx)

**Comment by tomershk** [comment]
> Hi @tomershk
> Can you please [sign your commits](https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md#sign-commits) so that the CI will run?
> Thanks

Will do.
What should I do with the current commit? how can I sign it?

**Comment by TalZa1810** [comment]
> sign your commits

The instructions are here:
https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md#sign-commits

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6474

File: src/GooglePreview/GooglePreview.js [hunk 1]
```diff
@@ -1,69 +1,93 @@
...
+  previewUrl: PropTypes.string,
+
+  /** A short description for the site */
+  description: PropTypes.string,
+
+  /** truncates text at a specific number of lines. */
+  descriptionMaxLines: PropTypes.number,
+
+  /** Widget background color. `transparent` will hide border as well*/
+  backgroundColor: PropTypes.oneOf(['white', 'transparent']),
```
[the remark this answers is not in the document]
Comment (tomershk) [reply src/GooglePreview/GooglePreview.js:85]: Agreed 😄

**Comment by tomershk** [comment]
Visual test updated to align with the updated look

**Review by EgoziE — APPROVED** [review]

**Pull request merged**